docs/how to/use-transferbench.rst

```diff
@@ -36,71 +39,102 @@ This simplifies to a simple copy operation when dealing with single SRC/DST.
 
 Three Executors are supported by TransferBench::
 
-   Executor:        SubExecutor:
-   1) CPU           CPU thread
-   2) GPU           GPU threadblock/Compute Unit (CU)
-   3) DMA           N/A.                                 (May only be used for copies (single SRC/DST)
+Executor:        SubExecutor:
```

Comment:
The formatting looks messed up:
![image](https://github.com/user-attachments/assets/f154021b-e002-42de-901a-9fd80b5b5a96)

Reply:
Should they be single lines

Reply:
Corrected.